hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -963,6 +963,12 @@ public SplitTableRegionResponse splitRegion(final RpcController controller,
       if (execController.getFailedOn() != null) {
         throw execController.getFailedOn();
       }
+
+      String remoteAddress = RpcServer.getRemoteAddress().map(InetAddress::toString).orElse("");
+      User caller = RpcServer.getRequestUser().orElse(null);
+      AUDITLOG.info("User {} (remote address: {}) master service request for {}.{}", caller,
+        remoteAddress, serviceName, methodName);
```

Comment:
Can we log more than methodName or perhaps methodName is what we usually put in audit log?

Reply:
methodName is what we usually put in audit log. We need to know what method is called by client when we are attacked!

Reply:
I was hoping for more than just method name if possible... like region name or table name if possible?

Reply:
I have checked the related code, there is no easy way to obtain infomration like region name or table name